languages/synchronize-wechat-zh_CN.po

```diff
@@ -0,0 +1,81 @@
+# Chinese translations for PACKAGE package
```

Comment:
Translation is based on the gettext project. http://www.gnu.org/software/gettext/
mo file should be generated from po file prior to code release. Because PHP outputs of the translated text is retrieved from mo file.

Reply:
type correction is here:
	    $return_obj = media_handle_sideload($fileArr, $postId);
            @unlink(tmpFile);
            @unlink($tmpFile);
            if (!is_wp_error($return_obj)) { // upload sucessfully
                return array('status_code' => $return_obj, 'err_msg' => 'upload successfully');
	    }

Reply:
Most of the code diff are trailing whitespace. My eclipse editor clean the trailing whitespace automatically.

Reply:
Trailing white spaces should be cleaned.